project/src/components/app/favorites/favorites.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import constant from '../../../constant';
+
+import Header from './../header/header.jsx';
+import FavoriteList from './favorite-list/favorite-list.jsx';
+import Footer from './footer/footer.jsx';
+
+function Favorites(props) {
+  const {hotels} = props;
+
+  const favoriteHotels = hotels.filter((obj) => obj.isFavorite === true);
```

Comment:
```suggestion
  const favoriteHotels = hotels.filter((obj) => obj.isFavorite);
```

Зачем ты булево значение сравниваешь с булевым значением?